x-pack/plugins/security_solution/public/management/links.test.ts

```diff
@@ -126,9 +127,26 @@ describe('links', () => {
   });
 
   describe('Host Isolation Exception', () => {
-    it('should NOT return HIE if `canReadHostIsolationExceptions` is false', async () => {
+    it('should return HIE if user has access permission (licensed)', async () => {
       (calculateEndpointAuthz as jest.Mock).mockReturnValue(
-        getEndpointAuthzInitialStateMock({ canReadHostIsolationExceptions: false })
+        getEndpointAuthzInitialStateMock({ canAccessHostIsolationExceptions: true })
+      );
+
+      const filteredLinks = await getManagementFilteredLinks(
+        coreMockStarted,
+        getPlugins(['superuser'])
```

Comment:
You don't need this `superuser` role for tests anymore. Consider updating the `getPlugins` function if you decide to remove all `superuser` roles in tests.